stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -772,11 +745,10 @@ internal constructor(
         members: List<Member> = emptyList(),
     ): Call<List<Member>> {
         logger.d { "[queryMembers] cid: $channelType:$channelId, offset: $offset, limit: $limit" }
-        val relevantPlugins = plugins.filterIsInstance<QueryMembersListener>().also(::logPlugins)
```

Comment:
We don't need to filter plugins anymore, because all of them implement those interfaces